stanford-sw/test/src/edu/stanford/ItemOnOrderTests.java

```diff
@@ -0,0 +1,120 @@
+package edu.stanford;
+
+import java.io.*;
+
+import org.junit.*;
+import org.marc4j.marc.ControlField;
+import org.marc4j.marc.DataField;
+import org.marc4j.marc.Leader;
+import org.marc4j.marc.MarcFactory;
+import org.marc4j.marc.Record;
+
+import edu.stanford.enumValues.Access;
+
+/**
+ * junit4 tests for Stanford University
+ *   cope properly with on order items - display, discoverable  INDEX-92
+ * @author Laney McGlohon
+ */
+public class ItemOnOrderTests extends AbstractStanfordTest
+{
+  private String AccfldName = "access_facet";
+  private final String onOrderFldVal = "On order";
+  private String LibfldName = "building_facet";
+
+  MarcFactory factory = MarcFactory.newInstance();
+  Leader ldr = factory.newLeader("01426cas a2200385Ia 4500");
+
+  DataField df999hloc = factory.newDataField("999", ' ', ' ');
+  {
+    df999hloc.addSubfield(factory.newSubfield('a', "XX"));
+    df999hloc.addSubfield(factory.newSubfield('k', "ON-ORDER"));
+    df999hloc.addSubfield(factory.newSubfield('m', "GREEN"));
+  }
+  DataField df999hcloc = factory.newDataField("999", ' ', ' ');
```

Comment:
I'm not sure I would both with changes now, but I would suggest that since each DataField is used exactly once for a single test below that this test data would be inline with the actual test. (I would leave factory and ldr up here, tho).